examples/upload-media-from-local/main.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+import os
+
+from utils.utils import PublicAPIClient, request_with_retry
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("files", nargs="+", help="path to the media file/folder")
```

Comment:
Sorry for nitpicking, but it's not "file/folder", but "file(s)". If you pass a folder, it won't work, the example passes a list of files (the shell expands the folder/* to a list of files).

Reply:
yeah I agree this is a better explanation for it. done